modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -60,6 +61,10 @@
      */
     public static final String CACHE_METRICS = "cache";
 
+    /** Histogram buckets for duration get, put, remove operations in nanoseconds. */
+    public static final long[] HISTOGRAM_BUCKETS = new long[] {1_000, 50_000, 75_000, 100_000, 250_000, 500_000,
```

Comment:
We should make default values smaller and fewer. Let's go with 100, 250, 500, 1000, 2500.

Reply:
These values are in nanoseconds.
Local results after 100k put,get,remove(10 threads):

Atomic Partitioned:
```
getHistogram=[0, 96928, 2750, 68, 61, 16, 20, 6, 27, 30, 40, 52, 2, 0]
putHistogram=[0, 0, 168, 6375, 59603, 27339, 2919, 1730, 1640, 147, 65, 14, 0, 0]
rmvHistogram=[0, 0, 166, 4492, 78386, 14869, 1123, 444, 426, 51, 12, 31, 0, 0]
```

Transactional Partitioned
```
getHistogram=[0, 99897, 27, 14, 17, 2, 4, 2, 3, 7, 14, 13, 0, 0]
putHistogram=[0, 0, 0, 0, 483, 32825, 37053, 10253, 18369, 612, 313, 76, 16, 0]
rmvHistogram=[0, 0, 0, 0, 385, 43341, 43206, 7196, 5585, 129, 64, 69, 25, 0]
```

I suggest adding `5_000` and `25_000` buckets instead of `1_000`, i.e.` 5_000, 25_000, 50_000, 75_000, 100_000, 250_000, 500_000, 750_000, 1_000_000, 3_000_000, 5_000_000, 10_000_000, 25_000_000, 60_000_000`

Then we get the following results:

Atomic Partitioned:
```
getHistogram=[3704, 81764, 10243, 3996, 86, 57, 7, 11, 7, 20, 23, 35, 40, 7, 0]
putHistogram=[0, 0, 0, 119, 5416, 64959, 21600, 3943, 2209, 1559, 112, 51, 32, 0, 0]
rmvHistogram=[0, 0, 0, 120, 5967, 81261, 10870, 966, 342, 342, 70, 29, 33, 0, 0]

```
Transactional Partitioned
```
getHistogram=[13503, 86302, 100, 20, 21, 23, 11, 0, 2, 5, 3, 4, 6, 0, 0]
putHistogram=[0, 0, 0, 0, 0, 563, 36118, 38115, 7766, 15880, 956, 475, 123, 4, 0]
rmvHistogram=[0, 0, 0, 0, 0, 381, 41211, 40955, 9767, 7012, 285, 260, 112, 17, 0]

```
WDYT?

Reply:
I don't want to have many number by defaults.
It seems too complicated for me.
3-4 buckets for "very fast, fast, acceptable, bad" cases looks more convenient for me.

Note, that we will measure cases with persistence enabled, also.
Note, that we will be able to configure histogram metric in runtime after #6696 will be merged.
Let's got with  1, 10, 100, 250, 1000 milliseconds.

Is it OK for you?